packages/manager/apps/dedicated/client/app/account/contacts/request/change/user-contacts-request-change.controller.js

```diff
@@ -8,14 +8,14 @@ angular
     '$translate',
     'UserAccount.services.Contacts',
     'Alerter',
-    'ducUser',
+    'User',
```

Comment:
it must be ducUser. Why have replaced with User./? you need to undo you changes and replace User with ducUser

Reply:
Fixed